src/test/java/unit/BqDatasetReferenced.java

```diff
@@ -339,6 +348,24 @@ void updateIndividualProperties() throws IOException {
         TestCommand.runAndParseCommandExpectSuccess(
             UFBqDataset.class, "resource", "describe", "--name=" + newName);
     assertEquals(newDescription, describeDataset.description);
+
+    updateDataset =
+        TestCommand.runAndParseCommandExpectSuccess(
+            UFBqDataset.class,
+            "resource",
+            "update",
+            "bq-dataset",
+            "--name=" + newName,
+            "--new-dataset-id=" + externalDataset2.getDatasetId());
+    assertEquals(externalDataset2.getDatasetId(), updateDataset.datasetId);
+    assertEquals(externalDataset2.getProjectId(), updateDataset.projectId);
```

Comment:
how come the project id also changed -- isn't the command above just updating the dataset id?

Reply:
it didn't change. externalDataset1 and 2 are both in the same project. just asserting it here to make sure that by not passing projectId here, the updateDatset don't have projectId null.